javagen/src/main/java/com/azure/autorest/util/TemplateUtil.java

```diff
@@ -59,6 +59,8 @@ public class TemplateUtil {
 
     public static final String DATE_UTC = "date-utc";
 
+    public static final String JSON_MERGE_PATCH_HELPER_CLASS_NAME = "JsonMergePatchHelper";
+
```

Comment:
move to `TemplateHelper` or `ClientModelUtil`

at present `TemplateUtil` is for the replacement of placeholder strings.